spec/reek/tree_dresser_spec.rb

```diff
@@ -1,7 +1,4 @@
-require_lib 'reek/cli/silencer'
-Reek::CLI::Silencer.silently do
-  require 'parser/ruby22'
-end
+require 'parser/ruby22'
```

Comment:
How about making this consistent to above and use ruby25 in regard to the good old boyscout rule? ;)